packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -43,7 +43,7 @@ export class ComponentUtils {
       folderName ? `${metadataType}_${folderName}` : metadataType
     }.json`;
     const componentsPath = path.join(
-      getRootWorkspacePath(),
+      workspaceUtils.getRootWorkspacePath(),
```

Comment:
same comment as before... is it OK to use a path with `.sfdx` here, or is there an API we should use?

Reply:
If we stick with sfdx, then to match the other code this should be added:
`import { SFDX_PROJECT_FILE } from '../constants';`